docs/design/logrotate.md

```diff
@@ -0,0 +1,73 @@
+# Ceph CSI Log Rotate Design Document
+
+Log Rotate is the ability to rotate the logs by controlling the size of log files.
+With csi it will be includes to all the csi pods which includes csi daemons and provisoners.
+Using logrotate will create a log file at the hostpath and enable the logrotator sidecar container.
+Logrotator sidecar container will make sure that log files are rotated as per the inputs.
```

Comment:
```suggestion
Log rotation involves managing log files by controlling their size. This feature will be added to the CephCSI and csi-addons containers in the CephCSI node-plugin and controller-plugin pods. When logrotate is used, it will generate a log file at the host path and run the logrotator sidecar container in CephCSI pods. The logrotator sidecar container will then ensure that log files are rotated based on predefined settings.
```

Reply:
Thanks it look much clear